src/Sylius/Bundle/CoreBundle/Fixture/Factory/TaxonExampleFactory.php

```diff
@@ -135,6 +140,9 @@ protected function configureOptions(OptionsResolver $resolver): void
             ->setAllowedTypes('translations', ['array'])
             ->setDefault('children', [])
             ->setAllowedTypes('children', ['array'])
+            ->setDefault('parent', function (Options $options): string {
```

Comment:
It seems wrong. We don't want to have the same taxon as it's parent by default. The default value should be null and we should add normalizer, which will translate some `code` to taxon itself. 
```suggestion
            ->setDefault('parent', null)
            ->setAllowedTypes('parent', ['null', 'string', TaxonInterface::class])
            ->setNormalizer('parent', LazyOption::findOneBy($this->taxonRepository, 'code'))
```